src/lexedata/importer/fromexcel.py

```diff
@@ -576,19 +573,22 @@ def load_dataset(
     EP.db.empty_cache()
     EP.parse_cells(lexicon_wb)
     # load cognate data set if provided by metadata
-    cognate = True if cognate_lexicon and dialect.cognates else False
-    if cognate:
-        try:
+    try:
+        dialect = argparse.Namespace(**dataset.tablegroup.common_props["special:fromexcel"])
```

Comment:
Wait, why again?

Reply:
I think, I changed it to this redundant form, because if a metadata-file does not contain any special:fromexcel-field it would raise a key-error. (e.g. the minimal test-set does not contain such a field). In the version before, I think, it would have thrown a key error